### PR TITLE
KF-2249 Visualisation Server Rock

### PR DESCRIPTION
**Pull request by misohu**
add visualisation-server rock

### CODE REVIEW

File: visualisation-server/rockcraft.yaml [hunk 1]
```diff
@@ -55,7 +56,7 @@ parts:
   
   non-root-user:
     plugin: nil
-    after: [mlflow]
+    after: [python, gcloud, files]
```
Comment (i-chvets) [inline visualisation-server/rockcraft.yaml:59]: Do we really need this?
I thought `non-root-user` part does not depend on those.

**Comment by i-chvets** [comment]
Filed to build this ROCK:
```
2023-06-08 09:58:29.053 :: 2023-06-08 13:58:28.590 :: + ln -s python3.6 /root/parts/python/install/usr/bin/python3            
2023-06-08 09:58:29.053 :: 2023-06-08 13:58:28.590 :: ln: failed to create symbolic link '/root/parts/python/install/usr/bin/python3': File exists                                                                                                           
2023-06-08 09:58:29.054 :: 2023-06-08 13:58:28.790 'override-build' in part 'python' failed with code 1.                      
2023-06-08 09:58:29.054 :: 2023-06-08 13:58:28.798 Traceback (most recent call last):                                         
2023-06-08 09:58:29.054 :: 2023-06-08 13:58:28.798   File "/snap/rockcraft/864/lib/python3.8/site-packages/rockcraft/parts.py", line 160, in run                                                                                                             
2023-06-08 09:58:29.054 :: 2023-06-08 13:58:28.798     aex.execute(action, stdout=stream, stderr=stream)                      
2023-06-08 09:58:29.054 :: 2023-06-08 13:58:28.798   File "/snap/rockcraft/864/lib/python3.8/site-packages/craft_parts/executor/executor.py", line 312, in execute                                                                                           
2023-06-08 09:58:29.054 :: 2023-06-08 13:58:28.798     self._executor.execute(actions, stdout=stdout, stderr=stderr)          
2023-06-08 09:58:29.054 :: 2023-06-08 13:58:28.798   File "/snap/rockcraft/864/lib/python3.8/site-packages/craft_parts/executor/executor.py", line 136, in execute                                                                                           
2023-06-08 09:58:29.054 :: 2023-06-08 13:58:28.798     self._run_action(act, stdout=stdout, stderr=stderr)                    
2023-06-08 09:58:29.054 :: 2023-06-08 13:58:28.798   File "/snap/rockcraft/864/lib/python3.8/site-packages/craft_parts/executor/executor.py", line 201, in _run_action                                                                                       
2023-06-08 09:58:29.054 :: 2023-06-08 13:58:28.798     handler.run_action(action, stdout=stdout, stderr=stderr)               
2023-06-08 09:58:29.054 :: 2023-06-08 13:58:28.798   File "/snap/rockcraft/864/lib/python3.8/site-packages/craft_parts/executor/part_handler.py", line 172, in run_action                                                                                    
2023-06-08 09:58:29.054 :: 2023-06-08 13:58:28.798     state = handler(step_info, stdout=stdout, stderr=stderr)               
2023-06-08 09:58:29.054 :: 2023-06-08 13:58:28.798   File "/snap/rockcraft/864/lib/python3.8/site-packages/craft_parts/executor/part_handler.py", line 309, in _run_build                                                                                    
2023-06-08 09:58:29.054 :: 2023-06-08 13:58:28.798     self._run_step(                                                        
2023-06-08 09:58:29.054 :: 2023-06-08 13:58:28.798   File "/snap/rockcraft/864/lib/python3.8/site-packages/craft_parts/executor/part_handler.py", line 485, in _run_step                                                                                     
2023-06-08 09:58:29.054 :: 2023-06-08 13:58:28.798     step_handler.run_scriptlet(                                            
2023-06-08 09:58:29.054 :: 2023-06-08 13:58:28.798   File "/snap/rockcraft/864/lib/python3.8/site-packages/craft_parts/executor/step_handler.py", line 278, in run_scriptlet                                                                                 
2023-06-08 09:58:29.054 :: 2023-06-08 13:58:28.799     raise errors.ScriptletRunError(                                        
2023-06-08 09:58:29.054 :: 2023-06-08 13:58:28.799 craft_parts.errors.ScriptletRunError: 'override-build' in part 'python' failed with code 1.              
```

Am I missing anything?

**Comment by misohu** [comment]
@i-chvets in my case no rpoblems with building `rockcraft 0.0.1.dev1`

**Comment by i-chvets** [comment]
> @i-chvets in my case no rpoblems with building `rockcraft 0.0.1.dev1`

That's exactly the version I am using and I get the error above.

**Comment by i-chvets** [comment]
Does not build with lxd 5.14

**Review by i-chvets — APPROVED** [review]
Approved.

**Pull request merged**